src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -154,7 +163,15 @@ public ExportConfiguration setPemFile(Path path) {
   }
 
   public boolean isPemFilePresent() {
-    return pemFile.isPresent();
+    if (pemFile.isPresent())  {
+      Path path = pemFile.get();
+      //Check if path is a file and is readable
+      if (Files.isRegularFile(path) && Files.isReadable(path)) {
```

Comment:
When you find yourself writing `if` blocks and using `Optional<T>` instances, odds are that you can do it better. In this case, you could use `Optional<T>.filter(Predicate<T>)` to encode those `if` blocks and then do a `map(path->true).orElse(false)`.